docs/guides/quick-start/01-introduction.md

```diff
@@ -0,0 +1,124 @@
+---
+stoplight-id: twpxvnspkg3yu
+---
+
+# Quick Start Guide
+
+## Overview
+
+Shopware Admin API allows web services to perform administrative tasks. This guide is designed to help you understand the Admin API, its endpoints, and its usage. You will get to learn everything about this API.
+
+* [Authentication as API user](01-introduction.md#authentication-and-setup)
+
+* [Product management](02-product-management.md)
+
+    * Product creation
+
+    * Categories
+
+    * Dynamic product groups
+
+    * Product reviews
+
+    * Cross-selling
+
+    * Price
+
+* [Media handling](03-media-handling.md)
+
+* [Document management](04-document-management.md)
+
+* [Order management](05-order-management.md)
+
+    * Create order
+
+    * Line item
+
+    * Order state handling
+    
+    * Refund payment
+
+* [CMS handling](06-cms-handling.md)
+
+    * Creation of layout
+
+    * Assignment of layout to a categoty
+
+It is useful to use an API client like Postman or Insomnia to follow this guide.
+
+## General
+
+The Admin API has a base route that is always relative to your Shopware instance host. Note that it might differ from your sales channel domain. Let us assume your Shopware host is
+
+https://shop.example.com/
```

Comment:
Code block? Or does that conflict with our guidelines?

Reply:
We can use code block here. I have updated it